kyo-prelude/shared/src/test/scala/kyo2/kernel/TraceTest.scala

```diff
@@ -120,7 +120,10 @@ class TraceTest extends Test:
                 val printWriter  = new PrintWriter(stringWriter)
                 ex.printStackTrace(printWriter)
                 printWriter.flush()
-                val trace = stringWriter.toString.linesIterator.takeWhile(!_.contains("anonfun")).mkString("\n")
+                val full   = stringWriter.toString
```

Comment:
This test was failing again because the execution flow changed. I've updated the logic to verify only frames introduce by Kyo.